errors.go

```diff
@@ -109,6 +109,10 @@ func (e _error) Format(s fmt.State, verb rune) {
 }
 
 // New returns an error with the supplied message.
+//
+// If New is only used function from this package in file processed
```

Comment:
// Be aware that tools like goimports may mistake this function for the standard
// libraries New function.